docs/user-guide/configure-map.md

```diff
@@ -0,0 +1,52 @@
+# Configure the map
+**********************
+
+With the Edit map configuration button <img src="../img/button/configure-map-button.jpg" class="ms-docbutton"/> the **Map Inline Editor** opens to give the opportunity to do quick customizations (like basic map settings, order of layers, layer opacity and something more) to the map (more advanced customizations then, are allowed only through the *Advanced Map Editor*).
+
+<img src="../img/configure-map/configure-map.jpg" class="ms-docimage"/>
+
+## Layers 
+
+The Map Inline Editor opens with the Layers section available, where it is possible to edit the layers settings (by selecting a layer in the TOC) and visibility of layers present in the map:
```

Comment:
The Map Inline Editor opens with the Layers section available, where it is possible to edit the layers settings (by selecting a layer in the TOC) and the visibility of layers present in the map: